app/assets/javascripts/application.js

```diff
@@ -76,7 +76,14 @@ function updateTotal(){
   $( ".total_value" ).text( total );
 }
 
-function itemExchange(id, char_id){
+function itemExchange(id, char_id, character_total, location_slug, store_slug){
```

Comment:
Identifier 'char_id' is not in camel case.<br>Identifier 'character_total' is not in camel case.<br>Identifier 'location_slug' is not in camel case.<br>Identifier 'store_slug' is not in camel case.